onnx/defs/printer.cc

```diff
@@ -216,6 +215,8 @@ void ProtoPrinter::print(const TensorProto& tensor, bool is_initializer) {
         output_ << "..."; // ParseData not instantiated for other types.
         break;
     }
+  } else if (tensor.has_data_location() && tensor.data_location() == TensorProto_DataLocation_EXTERNAL) {
```

Comment:
A minor suggestion: suggest moving this if condition above `if (tensor.has_raw_data())`. Technically, they are mutually exclusive conditions, but there are some situations, when converting data from internal to external format, where both conditions can be temporarily true. Better to output the external data in that case.

Reply:
Thank you for the technical detail. Done.